src/pages/git/git-rm/index.md

```diff
@@ -0,0 +1,23 @@
+## Git rm
```

Comment:
Hey there , thanks for the contribution 😄 You have to do some changes though , first you need to add something like ```
---
title: <title-name>
---
```
on the start of your article . 
Let me know when It's done, (or I'll try to do the change myself ) 👍 😄 Thanks again .

Reply:
Something like 
```
---
title: _title-name_ 
---
```

Reply:
Created new pull request